rllib/algorithms/ppo/ppo.py

```diff
@@ -405,7 +406,8 @@ def training_step(self) -> ResultDict:
             # that we don't have to do this back and forth
             # communication between driver and the remote
             # trainer workers
-
+            is_module_trainable = self.workers.local_worker().is_policy_to_train
+            self.learner_group.set_is_module_trainable(is_module_trainable)
```

Comment:
Maybe 
`self.learner_group.set_is_module_trainable(is_module_trainable if is_module_trainable else lambda *a, **b: False) `
?
I think the issues in CI are arising because `is_policy_to_train` is sometimes None...

Reply:
yeah, fixed it inside. I feel like there would be too much load on the algorithm developer end if they have to check whether is_policy_to_train is None or not. It's a very error prone design.